gms/gossiper.cc

```diff
@@ -757,6 +757,13 @@ future<> gossiper::update_live_endpoints_version() {
     });
 }
 
+future<std::set<inet_address>> gossiper::get_live_members_synchronized() {
+    auto live_members = gossiper::get_live_members();
+    return replicate_live_endpoints_on_change().then([live_members] {
```

Comment:
Please use coroutines:
```cpp
co_await replicate_live_endpoints_on_change();
co_return live_members;
```

It makes sense to not use coroutines in api/gossiper.cc because all of the code in that file is not using them, but here in gms/gossiper.cc there's no reason not to use coroutines.

Reply:
Changed in https://github.com/scylladb/scylladb/commit/3c7d2fa3f71645cccf1b313f41254e8659f38529